app/models/permission.rb

```diff
@@ -0,0 +1,29 @@
+class Permission
+  def initialize(user)
+    if user.nil?
+      allow 'home', [:index]
+      allow 'sessions', [:create]
+      allow 'api/version1_controller', [:get_proxy, :report_result]
```

Comment:
I wanted to check and make sure adding an individual key was something you still wanted.  Right now all the api version stuff works without authentication